assets/src/edit-story/utils/useFocusOut.js

```diff
@@ -19,7 +19,7 @@
  */
 import { useLayoutEffect } from 'react';
 
-function useFocusOut(ref, callback, deps) {
+function useFocusOut(ref, callback, deps, clickEvent = 'click') {
```

Comment:
Can we confirm that a focus is lost on pointerdown? If so, we should just make it default.

Reply:
Yeah, that was my question. It seems like a good default - I can't think of a case where you mouse-down outside the component and keep it open. Maybe if you were to drag something in - but we don't have a such a component as far as I can see actually.  I'm okay with making `pointerdown` default.

Reply:
When we implement #1041, we'll need to do something clever here, but other than that, I can't see a need to ever click outside.

Reply:
Found [this simple example](https://codepen.io/mudassir0909/pen/eIHqB) and did some manual checking. From what I can tell, "focusout" is fired right after the pointer down and before anything else. So I think this will be true to the order of the events if we just do pointer down.